sdk/servicebus/Microsoft.Azure.ServiceBus/src/Primitives/AzureActiveDirectoryTokenProvider.cs

```diff
@@ -5,59 +5,28 @@ namespace Microsoft.Azure.ServiceBus.Primitives
 {
     using System;
     using System.Threading.Tasks;
-    using Microsoft.IdentityModel.Clients.ActiveDirectory;
 
     /// <summary>
     /// Represents the Azure Active Directory token provider for the Service Bus.
     /// </summary>
     public class AzureActiveDirectoryTokenProvider : TokenProvider
     {
-        readonly AuthenticationContext authContext;
-        readonly ClientCredential clientCredential;
-#if !UAP10_0
-        readonly ClientAssertionCertificate clientAssertionCertificate;
-#endif
-        readonly string clientId;
-        readonly Uri redirectUri;
-        readonly IPlatformParameters platformParameters;
-        readonly UserIdentifier userIdentifier;
-
-        enum AuthType
-        {
-            ClientCredential,
-            UserPasswordCredential,
-            ClientAssertionCertificate,
-            InteractiveUserLogin
-        }
+        /// <summary>
```

Comment:
If I'm not overlooking something, this would seem to remove the dependency on the `Azure.IdentityModel` package, which does some very... interesting... things to hide concepts form some platforms.  Would love to see that ripped out of the project file and the target platforms updated to remove `uap10` in favor of `netstandard 1.4`